### PR TITLE
Removing QueryByResolver

### PR DESCRIPTION
**Pull request by carlosu7**
Also removing test since it used QueryByResolver and QueryByEmbedder commands.

`QueryByResolver ` was used to resolve project dependencies using Apache Maven Aether by using Maven Embedder (EmbedderFacade) to build and resolve project models, then extract the resolved dependencies and returns them as a list.

At this point, we would only have one available dependency command in the code base: QueryByParsing which is a SAFE dependency command.

### CODE REVIEW

**Comment by nahsra** [comment]
For each of these PRs, can you summarize what functionality we lose by removing this?

**Comment by aldrinleal** [comment]
If you're removing these perhaps better review the dependencies since they pull a lot of those

**Comment by carlosu7** [comment]
I'm removing all unused dependencies at the end in this PR: https://github.com/pixee/codemodder-java/pull/220

**Review by nahsra — APPROVED** [review]
